domains/DataCollection/Forms/SupplementaryForm/configs/envhealth.config.js

```diff
@@ -208,36 +274,57 @@ const configArray = {
     options: [
       {
         label: 'global.yes',
-        value: 'Yes'
+        value: 'Y'
       },
       {
         label: 'global.no',
-        value: 'No'
+        value: 'N'
```

Comment:
Don't think we should be changing the value of question. All others collected will not match those previously collected. Leave as No